compiler/src/iree/compiler/Codegen/Common/ReplaceSlowMinMaxOps.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 The IREE Authors
```

Comment:
This kind of works too, but I was thinking go one level deeper and just do the rewrite at LLVM dialect level? 

@Groverkss ?